pkg/scaffold/edit.go

```diff
@@ -51,29 +50,18 @@ func (s *editScaffolder) Scaffold() error {
 
 	// update dockerfile
 	if s.multigroup {
-		str, err = ensureExistAndReplace(
+		str = strings.Replace(
```

Comment:
It is really nice we proposed improvements for the code. however, it shows has no relation to the scope of this PR. Could you please do a PR specifically for that?